.travis.yml

```diff
@@ -1,65 +1,72 @@
-language: go
-go:
-- 1.14.x
 git:
   depth: 1
 notifications:
   email: false
+matrix:
+  include:
+    - language: R
+      r: 3.4
+      sudo: false
+      cache: packages
+      install:
+        - Rscript -e 'install.packages("lintr", dependencies = TRUE, repos="http://cran.rstudio.com/")'
+      script:
+        - make lint_R
+    
+    - language: go
+      go: 1.14.x
 
-install:
-  - sudo apt-get update
-  - sudo apt-get install r-base
-  - sudo apt-get install r-base-dev
-  - sudo Rscript -e 'install.packages("lintr", repos = "http://cran.us.r-project.org")'
-  
-env:
-  global:
-  - LANTERN_DBHOST=localhost
-  - LANTERN_DBPORT=5432
-  - LANTERN_DBUSER=lantern
-  - LANTERN_DBPASSWORD=postgrespassword
-  - LANTERN_DBUSER_READONLY=lantern_ro
-  - LANTERN_DBPASSWORD_READONLY=postgrespassword_ro
-  - LANTERN_DBUSER_READWRITE=lantern_rw
-  - LANTERN_DBPASSWORD_READWRITE=lantern_rw
-  - LANTERN_DBNAME=lantern
-  - LANTERN_DBSSLMODE=disable
-  - LANTERN_TEST_DBPASSWORD=postgrespassword
-  - LANTERN_TEST_DBNAME=lantern_test
-  - LANTERN_TEST_DBUSER=lantern
-  - LANTERN_TEST_DBPASSWORD=postgrespassword
-  - LANTERN_QUSER=capabilityquerier
-  - LANTERN_QPASSWORD=capabilityquerier
-  - LANTERN_QPORT=5672
-  - LANTERN_CAPQUERY_QNAME=capability-statements
-  - LANTERN_CAPQUERY_NUMWORKERS=10
-  - LANTERN_ENDPTQRY_NUMWORKERS=10
-  - LANTERN_CAPQUERY_QRYINTVL=1380
-  - LANTERN_TEST_QUSER=capabilityquerier
-  - LANTERN_TEST_QPASSWORD=capabilityquerier
-  - LANTERN_TEST_QNAME=test-queue
-  - LANTERN_TEST_ENDPTINFO_CAPQUERY_QNAME=test-endpoints-to-capability
-  - LINTR_COMMENT_BOT=false
+      env:
+        - LANTERN_DBHOST=localhost
+        - LANTERN_DBPORT=5432
+        - LANTERN_DBUSER=lantern
+        - LANTERN_DBPASSWORD=postgrespassword
+        - LANTERN_DBUSER_READONLY=lantern_ro
+        - LANTERN_DBPASSWORD_READONLY=postgrespassword_ro
+        - LANTERN_DBUSER_READWRITE=lantern_rw
+        - LANTERN_DBPASSWORD_READWRITE=lantern_rw
+        - LANTERN_DBNAME=lantern
+        - LANTERN_DBSSLMODE=disable
+        - LANTERN_TEST_DBPASSWORD=postgrespassword
+        - LANTERN_TEST_DBNAME=lantern_test
+        - LANTERN_TEST_DBUSER=lantern
+        - LANTERN_TEST_DBPASSWORD=postgrespassword
+        - LANTERN_QUSER=capabilityquerier
+        - LANTERN_QPASSWORD=capabilityquerier
+        - LANTERN_QPORT=5672
+        - LANTERN_CAPQUERY_QNAME=capability-statements
+        - LANTERN_CAPQUERY_NUMWORKERS=10
+        - LANTERN_ENDPTQRY_NUMWORKERS=10
+        - LANTERN_CAPQUERY_QRYINTVL=1380
+        - LANTERN_TEST_QUSER=capabilityquerier
+        - LANTERN_TEST_QPASSWORD=capabilityquerier
+        - LANTERN_TEST_QNAME=test-queue
+        - LANTERN_TEST_ENDPTINFO_CAPQUERY_QNAME=test-endpoints-to-capability
+        - LINTR_COMMENT_BOT=false
```

Comment:
This should probably be moved into the R lintr job unless you want to bring the lintr bot back!